git-multimail/git_multimail.py

```diff
@@ -3250,10 +3250,10 @@ def get_pusher_email(self):
             return super(GerritEnvironmentMixin, self).get_pusher_email()
 
     def get_fromaddr(self, change=None):
-        if self.__submitter and self.__submitter.find('<') != -1:
+        f = super(GerritEnvironmentMixin, self).get_fromaddr(change)
+        if not f and self.__submitter and self.__submitter.find('<') != -1:
             return self.__submitter
```

Comment:
Yeah, self.__submitter needs to take precedence over user.name/user.email for us.  I'm fine with multimailhook.from taking precedence over both, but we need something a bit more nuanced than this particular patch.